src/webapp/api.py

```diff
@@ -7,6 +7,8 @@
 
 from users import views as user_views  # type: ignore
 
+from .views import schema_view
```

Comment:
Make this an absolute import rather than relative i.e. `from webapp.views import schema_view`